tox.ini

```diff
@@ -27,12 +27,12 @@ deps = {[lint]deps}
 envdir = {toxworkdir}/lint
 
 [testenv:isort-check]
-commands = isort -rc -c settings_vial tests
+commands = isort -c settings_vial tests
```

Comment:
Due to https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/, we can just remove this